service/tbc/rpc_test.go

```diff
@@ -1692,11 +1510,11 @@ func indexAll(ctx context.Context, t *testing.T, tbcServer *Server) {
 
 	hash := bh.BlockHash()
 
-	if err := tbcServer.TxIndexer(ctx, &hash); err != nil {
+	if err = tbcServer.TxIndexer(ctx, &hash); err != nil {
 		t.Fatal(err)
 	}
 
-	if err := tbcServer.UtxoIndexer(ctx, &hash); err != nil {
+	if err = tbcServer.UtxoIndexer(ctx, &hash); err != nil {
```

Comment:
When using this idiom you really should keep `:=`.

Reply:
I think in most cases here, it is fine to set the value of `err` instead of shadowing it.

Reply:
ah yes will fix